### ISSUE TITLE
Android, installing a certificate to filter https with screenreader does not work

### ISSUE THREAD

**Issue by zstanecic** [AdguardTeam/AdguardForAndroid#4156]
### Prerequisites

Please answer the following questions for yourself before submitting an issue. **YOU MAY DELETE THE PREREQUISITES SECTION.**

- [ ] I am running the latest version - yes
- [ ] I checked the documentation and found no answer yes
- [ ] I checked to make sure that this issue has not already been filed yes

### Issue Details
i've tried to install certificate manually, as suggested by the app

samsung galaxy a71 android 12. adguard for android 3.6.8 and 4.0 nightly
- **AdGuard version:**
  - 
- **Filtering mode:**
  - 
- **Device:**
  - 
- **Operating system and version:**
  - 
- **Root access:**
  - 
root access no
### Expected Behavior
adguard  certificate should be installed

n/a
### Actual Behavior
n/a

n/a
### Screenshots
i cannot post screenshots, due to visual impairnment.

<details><summary>Screenshot:</summary>

</details>

**Comment by zstanecic** [comment]
Hi,

The path is what you said.

Then I was asked to select the file, and I get the error, that I need to install the certificate via the settings, not file storage manager.

**Comment by Birbber** [comment]
@zstanecic It came to our notice that Samsung released an update for 'My files' application. Could you check if the issue still reproducible?

**Comment by zstanecic** [comment]
Hi,

Bug is still reproducible.

**Comment by maxikuzmin** [comment]
@zstanecic try deleting the last "My files" update and go back to version 13.0.01.92
Some users have been helped by this solution. Perhaps in the future they will release an update with a solution to this problem. In any case, the problem is in "My Files" and not on our side.

**Comment by Birbber** [comment]
@zstanecic Have you tried it? Did it help?

**Comment by zstanecic** [comment]
Hi,

Unfortunately, this doesn’t work for my a71

**Comment by maxikuzmin** [comment]
@zstanecic is the question still relevant?  As far as we know, this problem was solved long ago on the Samsung side

**Comment by zstanecic** [comment]
Hi,

It’s relevant still on a71. It gives me an error that, I am installing from the storage, but not from the settings.

I’ve installed all updates available, and I have the latest fw for my Samsung.

**Comment by maxikuzmin** [comment]
@zstanecic can you reproduce this problem and take all the steps on video? Can you put it right here or email it to me? devteam@adguard.com

**Comment by zstanecic** [comment]
Hi,

Which program can i use to do it? I don’t know how to do it. It will be better to show you this somehow, via some share screen, as I use the screenreader to interact with the system.

**Comment by maxikuzmin** [comment]
@zstanecic simply swipe the shutter down and tap "Screen recorder". Repeat the steps and send the resulting video

![Screenshot_20220804-144429_One UI Home](https://user-images.githubusercontent.com/101641415/182811673-b1584f65-7335-4fab-a868-1e07aeefb486.jpg)

**Comment by maxikuzmin** [comment]
@zstanecic ping

**Comment by zstanecic** [comment]
pong. I have difficulty recording screenshots. this doesn't want to work in the backgroud on my phone.

**Comment by maxikuzmin** [comment]
@zstanecic see what action you have to take. Are you doing the same thing?


https://user-images.githubusercontent.com/101641415/183598688-f6c1cb91-90c5-4d07-b25b-a032e59fe82b.mp4

**Comment by maxikuzmin** [comment]
@zstanecic any news?

**Comment by zstanecic** [comment]
I cannot read your pictures. I am blind and using a screenreader, but but the bug persists.

**Comment by maxikuzmin** [comment]
@zstanecic please follow these steps when setting up HTTPS filtering: 

1. open AdGuard Settings
2. open the Network section
3. select HTTPS Filtering 
4. move the slider from OFF to ON

You then follow the open instructions:

1. On the "What is HTTPS filtering" page, read and click "Next"
2. On the "What is HTTPS filtering secure" page, read and click on "Next
On the "AdGuard Certificate" page, please read and click on "Save it now".

This will take you to the "Downloads" section of the system. 
scroll down to the "Save" button (The name of the certificate you are saving will be next to this button, remember the name). Click on the "Save" button.

Then the "How to install certificate" window will appear.
press "Open settings".

this will open your system settings 

1. Click on the magnifying glass (Search) 
2. type in "CA certificate" in the search box

the "Install from device storage" setting will appear

1. Click on "CA certificate".

The message "Your privacy is at risk" will appear

2. Listen and click on "Install anyway", confirm with your password

3. navigate to "Download" folder (usually the files are saved there automatically)
4. choose your certificate from the list of files you have saved previously (usually the first file in the list if you have sorted it by date)
5. when you have selected your certificate, click on the "Done" button at the bottom of the screen.
Done. Now you have HTTPS filtering set up, you can go back to AdGuard and use the application

Please let us know if everything worked for you

**Comment by zstanecic** [comment]
Now it worked!
Thanks. The short guide was not so explicit, when it comes to say, that the box should be ticked and clicked the done button.
I have installed this.

**Comment by maxikuzmin** [comment]
@zstanecic great! Good luck!

**Comment by zstanecic** [comment]
Please, don’t communicate with me using images. I am blind and using a screenreader.

**Comment by zstanecic** [comment]
It doesn’t allow recording in the background.

Do you have other solution?
